aea/helpers/dialogue/base.py

```diff
@@ -679,18 +736,6 @@ def update(self, message: Message) -> Optional[Dialogue]:
                 dialogue_reference=dialogue_reference,
                 role=self.role_from_first_message(message),
             )  # type: Optional[Dialogue]
-        elif (  # new dialogue by self
```

Comment:
Cannot be removed; we want to be compatible and also see my other comment above

Reply:
Done